Parsers/ASim/Watchlists/ASimDisabledParsers.json

```diff
@@ -11,12 +11,12 @@
     },
       "resources": [
           {
-          "name": "[concat(parameters('workspaceName'), '/Microsoft.SecurityInsights/ASimDisabledParser')]",
+          "name": "[concat(parameters('workspaceName'), '/Microsoft.SecurityInsights/ASimDisabledParsers')]",
           "type": "Microsoft.OperationalInsights/workspaces/providers/Watchlists",
           "kind": "",
           "properties": {
-              "displayName": "ASim Disabled Parser",
-              "source": "ASimDisabledParser.csv",
+              "displayName": "ASim Disabled Parsers",
+              "source": "ASimDisabledParsers.csv",
```

Comment:
for the same reason as above, this has no functional impact, and a change might. I prefer not to make that change at the moment,